devices/common_patches/h7_exti_dualcore.yaml

```diff
@@ -0,0 +1,853 @@
+          description: CPU2 event mask on Event input x
+          bitOffset: 07
+          bitWidth: 1
+        MR8:
+          description: CPU2 event mask on Event input x
+          bitOffset: 08
+          bitWidth: 1
+        MR9:
+          description: CPU2 event mask on Event input x
+          bitOffset: 09
+          bitWidth: 1
+        MR10:
+          description: CPU2 event mask on Event input x
+          bitOffset: 10
+          bitWidth: 1
+        MR11:
+          description: CPU2 event mask on Event input x
+          bitOffset: 11
+          bitWidth: 1
+        MR12:
+          description: CPU2 event mask on Event input x
+          bitOffset: 12
+          bitWidth: 1
+        MR13:
+          description: CPU2 event mask on Event input x
+          bitOffset: 13
+          bitWidth: 1
+        MR14:
+          description: CPU2 event mask on Event input x
+          bitOffset: 14
+          bitWidth: 1
+        MR15:
+          description: CPU2 event mask on Event input x
+          bitOffset: 15
+          bitWidth: 1
+        MR16:
+          description: CPU2 event mask on Event input x
+          bitOffset: 16
+          bitWidth: 1
+        MR17:
+          description: CPU2 event mask on Event input x
+          bitOffset: 17
+          bitWidth: 1
+        MR18:
+          description: CPU2 event mask on Event input x
+          bitOffset: 18
+          bitWidth: 1
+        MR19:
+          description: CPU2 event mask on Event input x
+          bitOffset: 19
+          bitWidth: 1
+        MR20:
+          description: CPU2 event mask on Event input x
+          bitOffset: 20
+          bitWidth: 1
+        MR21:
+          description: CPU2 event mask on Event input x
+          bitOffset: 21
+          bitWidth: 1
+        MR22:
+          description: CPU2 event mask on Event input x
+          bitOffset: 22
+          bitWidth: 1
+        MR23:
+          description: CPU2 event mask on Event input x
+          bitOffset: 23
+          bitWidth: 1
+        MR24:
+          description: CPU2 event mask on Event input x
+          bitOffset: 24
+          bitWidth: 1
+        MR25:
+          description: CPU2 event mask on Event input x
+          bitOffset: 25
+          bitWidth: 1
+        MR26:
+          description: CPU2 event mask on Event input x
+          bitOffset: 26
+          bitWidth: 1
+        MR27:
+          description: CPU2 event mask on Event input x
+          bitOffset: 27
+          bitWidth: 1
+        MR28:
+          description: CPU2 event mask on Event input x
+          bitOffset: 28
+          bitWidth: 1
+        MR29:
+          description: CPU2 event mask on Event input x
+          bitOffset: 29
+          bitWidth: 1
+        MR30:
+          description: CPU2 event mask on Event input x
+          bitOffset: 30
+          bitWidth: 1
+        MR31:
+          description: CPU2 event mask on Event input x
+          bitOffset: 31
+          bitWidth: 1
+      C1PR1:
```

Comment:
Indentation is wrong here, dedent C1PR1?

Reply:
Fixed, along with register names